khmer/khmer_args.py

```diff
@@ -356,25 +361,29 @@ def add_loadgraph_args(parser):
                         help='load a precomputed k-mer graph from disk')
 
 
-def calculate_graphsize(args, graphtype, multiplier=1.0):
+def calculate_graphsize(args, graphtype, multiplier=1.0,
+                        sysmem=psutil.virtual_memory().available):
```

Comment:
Yeah, that's a bit cleaner. And I can also avoid cluttering the `create_*graph` functions by testing `calculate_graphsize` directly. That'll also help avoid problems with Jenkins like when I, oh, tried to create a 64 GB countgraph for testing! :-/